src/ray/rpc/grpc_server.cc

```diff
@@ -107,10 +107,9 @@ void GrpcServer::Run() {
   // Create calls for all the server call factories.
   for (auto &entry : server_call_factories_) {
     for (int i = 0; i < num_threads_; i++) {
-      // Create a buffer of 100 calls for each RPC handler.
-      // TODO(edoakes): a small buffer should be fine and seems to have better
-      // performance, but we don't currently handle backpressure on the client.
-      int buffer_size = 100;
+      // When there is no max active RPC limit, a call will be added to the completetion
+      // queue before processing starts, so adding only 1 call is enough.
+      int buffer_size = 1;
```

Comment:
I'm not very familiar with this. But does this mean  we'll accept request slower than before? For example, accept the request 1by1 because the size ie only 1 and only after we do the actual processing, a new one will be added.

Reply:
The nightly tests do not show a big difference, but from reading the code of completion queue, I can see that having a number of tags buffered can make processing more efficient. Adjusted the value to be closer to the existing value.